setlexsem/generate/sample.py

```diff
@@ -105,6 +105,8 @@ def __init__(self, m_A: int, m_B: int, item_len=None, random_state=None):
         self.m_A = m_A
         self.m_B = m_B
         self.item_len = item_len
+        if isinstance(self.item_len, str):
+            self.item_len = eval(self.item_len)
```

Comment:
Why `eval` instead of `int`?

Reply:
item_len is None (string), we want to convert it to None object, hence, we use eval here.

Reply:
Gotcha. Thanks.